v2/thirdparty/common-customizations/multi-tenancy/custom-provider.mdx

```diff
@@ -326,7 +326,7 @@ async function createTenant() {
             scope: ["email", "profile"]
         }],
         // highlight-start
-        oidcDiscoveryEndpoint: "https://example.com",
+        oidcDiscoveryEndpoint: "https://example.com/.well-known/openid-configuration",
```

Comment:
there is text around this snippet that says that we do not add the wellknown path. That needs to be removed.